testdata/scripts/convert_fileoptions.txt

```diff
@@ -0,0 +1,25 @@
+env HOME=$WORK/home
+
+gunk convert util.proto
+cmp util.gunk util.gunk.golden
+
+-- util.proto --
+syntax = "proto3";
+
+package util;
+
+option optimize_for = 2;
+
+option java_package = "com.example.util"
+option java_outer_classname = "JavaOuterClassname"
+
+-- util.gunk.golden --
+// +gunk file.OptimizeFor(2)
```

Comment:
@mvdan If we wanted to output `file.OptimizeFor(file.CodeSize)` (https://github.com/gunk/opt/blob/master/file/file.gunk#L12), do you whether that is easy or not?

Reply:
Should be reasonable to do. I'll have a look.